pkg/operator/staticpodcontroller/interfaces.go

```diff
@@ -0,0 +1,37 @@
+package staticpodcontroller
+
+import (
+	operatorv1alpha1 "github.com/openshift/api/operator/v1alpha1"
+	"k8s.io/client-go/tools/cache"
+)
+
+type OperatorClient interface {
+	Informer() cache.SharedIndexInformer
+	Get() (*operatorv1alpha1.OperatorSpec, *StaticPodConfigStatus, string, error)
+	UpdateStatus(string, *StaticPodConfigStatus) (*StaticPodConfigStatus, error)
+}
+
+// TODO move to openshift/api I think
```

Comment:
Why? IIUC this is only for kube-apiserver, is it not?

Reply:
> Why? IIUC this is only for kube-apiserver, is it not?

apiserver, controller, scheduler, possibly etcd.